nan_implementation_pre_12_inl.h

```diff
@@ -185,7 +186,8 @@ Factory<v8::String>::New(const uint8_t * value, int length) {
   if (wideString.size() == 0) {
     return v8::String::Empty();
   } else {
-    return v8::String::New(&wideString.front(), wideString.size());
+    return v8::String::New(&wideString.front()
+         , static_cast<int>(wideString.size()));
```

Comment:
I think there's an assert in the widenString function called above, so it should not be necessaey to assert again, as the length ought to have been checked already.

On January 18, 2015 11:46:37 PM EET, Ben Noordhuis <notifications@github.com> wrote:
>> @@ -185,7 +186,8 @@ Factory<v8::String>::New(const uint8_t * value,
>int length) {
>>    if (wideString.size() == 0) {
>>      return v8::String::Empty();
>>    } else {
>> -    return v8::String::New(&wideString.front(), wideString.size());
>> +    return v8::String::New(&wideString.front()
>> +         , static_cast<int>(wideString.size()));
>
>No assert here?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/rvagg/nan/pull/247/files#r23137337